django/webscraper/web_scraper.py

```diff
@@ -104,7 +124,8 @@ def get_links_to_all_animal(href: str) -> set[str]:
     result: set[str] = set()
     page_iter: int = 1
     while True:
-        print(f"fetching from {page_iter}")
+        logging.info("fetching from %lu\n", page_iter)

```

Comment:
Logging addition is awesome

Reply:
Consider logging page id / path whatever not just its id.
Estimate of how much is left would be welcomed